amarachukwu-agbo/random-number-generator
# #165145088 Set up jest to run tests

#### What does this PR do?
Sets up jest for testing

#### Description of Task to be completed?
Install `jest` and create test command

#### How should this be manually tested?
Clone the repo and install application dependencies. Run command  `yarn test`. You should see tests running.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#165145088](https://www.pivotaltracker.com/story/show/165145088)

#### Screenshots (if appropriate)
N/A

#### Questions: 
N/A
